adapters/adnuntius/adnuntius.go

```diff
@@ -202,6 +200,13 @@ func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters
 			})
 	}
 
+	endpoint, err := makeEndpointUrl(ortbRequest, a, noCookies)
+	if err != nil {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("failed to parse URL: %s", err),
+		}}
+	}
```

Comment:
I noticed this error case isn't covered by tests, would it be possible to update your tests for this fail to parse URL case?

Reply:
Sure, will add that.

Reply:
I couldn't find a way to trigger this, so I removed the check.

Reply:
I see, I think we should still be handling the error regardless. So even if you can't cover it for a test, we still want to have the error check if you can re-add it please.